### DISCUSSION TITLE
Adding Custom components

### DISCUSSION THREAD

**Discussion by htihaS** [LiveDuo/destack#117] [Q&A]
Can i Add my own component package? if yes how?

**Comment by LiveDuo** [comment]
@htihaS Currently you have to fork the library or use [patch package](https://www.npmjs.com/package/patch-package). If you want to try that out, themes are defined [here](https://github.com/LiveDuo/destack/blob/main/lib/client/vanilla/editor.tsx#L32) and theme data should look like [this](https://github.com/LiveDuo/destack/tree/main/lib/themes).
